Rephrase README.md (#451)
NCAR/geocat-comp: README.md

GeoCAT-comp is the computational component of the

> [GeoCAT](https://geocat.ucar.edu/) project. eoCAT-comp provides computational

Reviewer: ```suggestion
[GeoCAT](https://geocat.ucar.edu/) project. GeoCAT-comp provides computational
```

Author: Ah, good catch, thanks!

> [GeoCAT](https://geocat.ucar.edu/) project. eoCAT-comp provides computational

Reviewer 2: ```suggestion
[GeoCAT](https://geocat.ucar.edu/) project. GeoCAT-comp provides computational
```

GeoCAT-comp is the computational component of the

> [GeoCAT](https://geocat.ucar.edu/) project. GeoCAT-comp provides implementations
> of computational functions for operating on geosciences data. Many of these

Reviewer: ```suggestion
[GeoCAT](https://geocat.ucar.edu/) project. GeoCAT-comp provides computational functions for operating on geosciences data. Many of these
```

> functions originated in NCL and were translated into Python with the help of
> GeoCAT-comp; however, developers are welcome to come up with novel computational
> functions for geosciences data.

Reviewer: ```suggestion
functions were originally implemented in the NCAR Command Language (NCL) and were ported to Python; however, developers are welcome to suggest other novel computational functions for geosciences data.
```

Author: I don't love the "ported to python" wording here. Additionally, I feel like "originated in NCL" is more technically correct since the functions are from NCL, but that's not necessarily the methods' very first implementation.

I did a smaller reword for the first part and taken the "come up with" -> "suggest" wording.

How do you feel about the version I just committed?
